### DISCUSSION TITLE
Computer time, Silicon production model for instance

### DISCUSSION THREAD

**Discussion by Mskater** [deepmodeling/deepmd-kit#757] [Q&A]
Hello,
  Is it possible, or not really practical to obtain a production model of, say, the Silicon data
on a Ubuntu 18.04, with CPU version?
   sel: [432] and stop_batch: 1000000,  I calculate to take about 10 days, and that may not be enough?
In general, where is the point one should put this on a cluster?  And how many processors needed
for decent expedience?
Thanks very much for insight!

**Comment by njzjz** [comment]
How many cores did you use? 10 days is a common cost if you only use a few processors.

Anyway, it is recommended to use GPUs to get the fastest computation.
